src/profile_manager/templates/profile_manager/snippets/staff_list.html

```diff
@@ -0,0 +1,121 @@
+            Admin
+        </th>
+        {% endif %}
+
+        </tr>
+    </thead>
+
+    <tbody>
+        {% for object in object_list %}
+
+        <tr {% if request.user.is_staff %}
+                {% if object.not_earning_xp or object.banned_from_comments %}
+                    class="warning"
+                {% elif object.gone_stale %}
+                    class="danger"
+                {% endif %}
+            {% endif %}>
+
+            <td><img class="icon-md img-rounded" src="{{ object.get_avatar_url }}" alt="avatar"/></td>
+            <td>{{ object.user.first_name }}</td>
+            <td>{{ object.get_preferred_name }}</td>
+            <td>{{ object.user.last_name|default:"-" }}</td>
+            <td>{{ object.alias_clipped|default:"-" }}</td>
+
+            <td>{% if object.user.portfolio.listed_locally or request.user.is_staff %}
+                    <a href="{{ object.user.portfolio.get_absolute_url }}">View</a>
+                {% else %}<i class="fa fa-eye-slash"></i>{% endif %}
+            </td>
+
+            {% if request.user.is_staff %}
+                <td>
+                    {{ object.user.last_login|date:'y/m/d' }}<br>
+                    <small>{{ object.user.last_login|timesince }} ago</small>
+                </td>
+                <!-- `last_login_datetime` -->
+                <td>
+                    {{ object.user.last_login|date:"y/m/d" }} {{ object.user.last_login|time:"H:i:s"}}
+                </td>
+                <!-- end `last_login_datetime` -->
+                <td><a href="{% url 'profiles:profile_detail' object.id %}">{{ object.user.username|truncatechars:14 }}</a></td>
+
+                {#      Admin buttons #}
+                <td>
+                    <div class="btn-group action-btn-group">
+                        <a href="{% url 'profiles:comment_ban_toggle' object.id %}"
+                            title="{% if object.banned_from_comments %}Remove comment ban
+                        {% else %}Ban student from public comments{% endif %}"
+                            class="btn btn-sm {% if object.banned_from_comments %}btn-success{% else %}btn-default{% endif %}">
+                            <i class="fa fa-commenting-o"></i>
+                        </a>
+                        <a href="{% url 'profiles:xp_toggle' object.id %}"
+                            title="{% if object.not_earning_xp %}Turn quest skipping off
+                        {% else %}Allow user to skip quests. Gives user a button allowing them skip quests without approval. Skipped quests count as completed for prerequisite purposes but don't grant XP.{% endif %}"
+                            class="btn btn-sm
+                    {% if object.not_earning_xp %}btn-success{% else %}btn-default{% endif %}">
+                            <i class="fa fa-magic"></i>
+                        </a>
+                        <a href="{% url 'profiles:change_password' object.user.pk %}" class="btn btn-sm btn-default"
```

Comment:
Hmmm... since we have some logic that prevents staff from changing other staff passwords (I think?) maybe remove this too, can already access from the profile as needed. No need for the quick button here..

Reply:
removed